server.py

```diff
@@ -25,7 +25,7 @@ class GameServer(object):
     def __init__(self, port=9876):
         self.connect = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         self.connect.bind(('0.0.0.0', port))
-        self.clientAddr = None
+        self.clientAddr = ('127.0.0.1', port)
```

Comment:
I just want to keep it None here. :p

Reply:
Okay, It would works well if "self.clientAddr" is set as "None". It doesn't need to be changed.
However, I think that it would be clear in that way.